.github/workflows/e2e.yml

```diff
@@ -0,0 +1,57 @@
+name: Turbinia E2E Test
+
+on: [push, pull_request]
+
+jobs:
+  e2e-test:
+    name: Run local stack e2e test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Pull docker images for cache
+      run: |
+        docker pull ubuntu:18.04
+        docker pull us-docker.pkg.dev/osdfir-registry/turbinia/release/turbinia-server-dev:latest
+        docker pull us-docker.pkg.dev/osdfir-registry/turbinia/release/turbinia-worker-dev:latest
+    - name: Build Turbinia server docker image
+      run: docker build --cache-from=ubuntu:18.04,us-docker.pkg.dev/osdfir-registry/turbinia/release/turbinia-server-dev:latest -t turbinia-server-dev -f docker/server/Dockerfile .
+    - name: Build Turbinia server worker image
+      run: docker build --cache-from=ubuntu:18.04,us-docker.pkg.dev/osdfir-registry/turbinia/release/turbinia-worker-dev:latest -t turbinia-worker-dev -f docker/worker/Dockerfile .
+    - name: Prepare Turbinia configuration and e2e evidence
+      run: |
+        mkdir ./conf
+        mkdir ./evidence
+        chmod 777 ./evidence
+        sed -f ./docker/local/local-config.sed ./turbinia/config/turbinia_config_tmpl.py > ./conf/turbinia.conf
+        curl https://raw.githubusercontent.com/obsidianforensics/hindsight/master/tests/fixtures/profiles/60/History > History
+        tar -vzcf ./evidence/history.tgz History
+    - name: Verify Turbinia configuration and evidence
+      run: |
+        cat ./conf/turbinia.conf
+        ls -al ./evidence
+    - name: Startup local turbinia docker-compose stack
+      run: docker-compose -f ./docker/local/docker-compose.yml up -d
+    - name: Sleep for 5s
+      uses: juliangruber/sleep-action@v1
+      with:
+        time: 5s
+    - name: Show running instances
+      run: docker ps -a
+    - name: Show container logs
+      run: |
+        docker logs turbinia-server
+        docker logs turbinia-worker
+    - name: Run E2E test
+      run: |
+        docker exec -t turbinia-server turbiniactl compresseddirectory -l /evidence/history.tgz > request.txt
+        echo "REQUEST_ID=`cat request.txt |grep Creating | sed -e 's/.*request \(.*\) with.*/\1/'`" >> $GITHUB_ENV
+    - name: Sleep for 60s
+      uses: juliangruber/sleep-action@v1
+      with:
+        time: 60s
+    - name: Poll request status
+      run: docker exec turbinia-server turbiniactl -a status -r $REQUEST_ID
```

Comment:
Should we check the status of this for a report file to make sure it was successful?  We can use the `turbiniactl -a status --dump_json -r $REQUEST_ID` flags to get it in json output.

Reply:
Alternatively we could also look in the output dir to find the resulting report file.

Reply:
Tried that but the json output is bodged. I'll open a bug to fix the json output of both the status command and the request create command. 
eg the status command json output contains below
```
'last_update': datetime.datetime(2021, 2, 4, 9, 37, 32, 357199)
```

Reply:
https://github.com/google/turbinia/issues/712

Reply:
Cool, thanks for fixing the json output stuff.   Another thing I just remembered is that you can actually supply the request ID on the command line too, and the client will pass that along with the request instead of generating a new one.